src/Workspaces/CSharp/Portable/Utilities/CSharpUseExplicitTypeHelper.cs

```diff
@@ -88,13 +88,20 @@ protected override bool ShouldAnalyzeForEachStatement(ForEachStatementSyntax for
                 typeName.Parent.Parent.IsKind(SyntaxKind.LocalDeclarationStatement, SyntaxKind.ForStatement, SyntaxKind.UsingStatement))
             {
                 // check assignment for variable declarations.
-                var variable = ((VariableDeclarationSyntax)typeName.Parent).Variables.First();
+                var variableDeclaration = (VariableDeclarationSyntax)typeName.Parent;
+                var variable = variableDeclaration.Variables.First();
                 if (!AssignmentSupportsStylePreference(
                         variable.Identifier, typeName, variable.Initializer.Value,
                         semanticModel, optionSet, cancellationToken))
                 {
                     return false;
                 }
+
+                // This is handled by another code fix.
```

Comment:
nit: mention the name of hte code fix that fixes this?  you might also want to explain that this is an error case, and that's why we don't think the main user feature is the right codepath to handle this sort of conversoin.

Reply:
➡️ Fixed in c0b9ac255eda3831499d11ff335a731348c8bf0b